src/App.test.js

```diff
@@ -1,8 +1,84 @@
+import renderer from 'react-test-renderer';
```

Comment:
- [ ] One of your tests is failing, kindly fix it:
![Screenshot from 2022-03-23 22-38-36](https://user-images.githubusercontent.com/62705947/159800533-dddb90c1-042f-44b7-956f-bcec1c31e07f.png)